src/Microsoft.Health.Fhir.Azure/ExportDestinationClient/AzureExportDestinationClient.cs

```diff
@@ -119,9 +119,66 @@ public async Task CommitAsync(CancellationToken cancellationToken)
         {
             CheckIfClientIsConnected();
 
+            try
+            {
+                await Policy.Handle<StorageException>()
+                    .WaitAndRetryAsync(
+                        retryCount: 2,
+                        sleepDurationProvider: (retryCount) => TimeSpan.FromSeconds(5 * (retryCount - 1)),
```

Comment:
In other places in the code we have an 'exponential retry' policy defined as `.WaitAndRetryAsync(3, retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)))`, so the delay starts shorter and backs off exponentially, this one looks linear.
Perhaps we could create an extension method on `PolicyBuilder` that allows us to reference the pattern by name (ExponentialBackoffRetry, LinearBackoffRetry)?